src/main/java/com/bootcamp/demo/service/BookingService.java

```diff
@@ -0,0 +1,24 @@
+package com.bootcamp.demo.service;
+
+import com.bootcamp.demo.models.Booking;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Logger;
+
+public class BookingService implements  IBookingService{
+    private static final Logger LOGGER = Logger.getLogger(BookingService.class.getName());
```

Comment:
We use the `Slf4j` logger. Please update it accordingly. Checkout in other PRs. Ypu need to add the `org.slf4j.api `dependency